blog/docs/articles/highlighting-value-knative-c-suite.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Highlighting the Value of Knative for the C-Suite"
+linkTitle: "Highlighting the Value of Knative for the C-Suite"
+date: 2021-11-04
+description: "Deploy faster and more cost-effectively without hard-to-find, specialized expertise."
+type: "blog"
+---
+
+# Highlighting the Value of Knative for the C-Suite
+
+<table style="border: 0;">
+<tr style="background-color: var(--md-default-bg-color);">
+<td style="border: 0;" width="33%">
+
+<div style="padding:50px;">
+<em style="color:#0071c7;font-size: 1em;">"What we are doing with Knative is to provide a developer experience that makes it easier to focus on code. Cloud,developers focus on the business problems they are solving without having to coordinate or wait for approvals from platform teams to scale their apps.. Knative is a framework that helps automate platform capabilities so your apps can scale as if they were running on Serverless compute."</em>
```

Comment:
```suggestion
<em style="color:#0071c7;font-size: 1em;">"What we are doing with Knative is to provide a developer experience that makes it easier to focus on code. Cloud developers focus on the business problems they are solving without having to coordinate or wait for approvals from platform teams to scale their apps.. Knative is a framework that helps automate platform capabilities so your apps can scale as if they were running on Serverless compute."</em>
```

Reply:
I suggest making the quotes more visually separate from the other content, such as how the quotes like _"Knative gives us a foundation of consistency," said Andrew Webber, Senior Software Engineer._ appears in https://knative.dev/docs/about/case-studies/deepc/